contracts/sdjwt-verifier/Cargo.toml

```diff
@@ -15,31 +15,25 @@ crate-type = ["cdylib", "rlib"]
 doctest = false
 
 [features]
-backtraces = ["cosmwasm-std/backtraces"]
 library = []
-migration = []
-mt = ["sylvia/mt"]
 
 [dependencies]
 avida-common= { path = "../../packages/common/"}
 avida-cheqd= { path = "../../packages/cheqd/"}
+cosmwasm-schema = { workspace = true }
 cosmwasm-std = { workspace = true }
 cw-storage-plus = { workspace = true }
-cosmwasm-schema = { workspace = true }
-sylvia = { workspace = true }
-sylvia-derive = { workspace = true }
-serde = {workspace = true}
-serde-json-wasm = {workspace = true}
 cw2 = { workspace = true }
-cw-utils= { workspace = true }
-sd-jwt-rs = { git = "https://github.com/nymlab/sd-jwt-rust", rev = "c05b255790a6b2f702e", default-features = false, features = ["no_rand"]}
-jsonwebtoken = { git = "https://github.com/nymlab/jsonwebtoken", rev = "98763b38713c54f", default-features = false, features = ["no_rand"] }
+schemars = { workspace = true }
+serde = { workspace = true }
 thiserror = { workspace = true }
-serde_json = {version = "1.0.116", default-features = false, features = ["alloc"]}
-
+sd-jwt-rs = { workspace = true }
+jsonwebtoken = { workspace = true }
+serde_json = { workspace = true, default-features = false, features = ["alloc"]}
+serde-json-wasm = {workspace = true}
```

Comment:
I saw this 
https://github.com/CosmWasm/cosmwasm/issues/2195

looks like we can replace  remove it?

Reply:
Sure, https://github.com/nymlab/AVIDA/pull/19/commits/f8166566cf2d12d924bbb4ea62706a29019518c5